src/kOS/Module/kOSLightModule.cs

```diff
@@ -28,23 +28,24 @@ internal class kOSLightModule : PartModule
 
         [KSPField(isPersistant = false, guiActive = false)]
         public string animationName = "";
-        
+
         [KSPField(isPersistant = false, guiActive = false)]
         public string pulseWidth = "";
 
         [KSPField(isPersistant = false, guiActive = false)]
         public string gapWidth = "";
 
         private double lastTime = double.MaxValue;
+        private Color lastColor = new Color(0, 0, 0, 0);
 
         private readonly Color powerOffColor = new Color(0, 0, 0, 0);
 
         private ModuleLight lightModule;
-        private Light[] lights;
-        private Renderer[] renderers;
+        private List<Light> lights;
+        private List<Renderer> renderers;
```

Comment:
It's our own definition for the field, but Squad changed the type of the methods we use to populate the field (I have no idea why Animations didn't get changed too, see the next line).  We could have continued to re-cast back to an array, but at that point it's a coin flip which is better on our end.